diff --git crater-website/content/docs/user/quick-start/login.jp.mdx crater-website/content/docs/user/quick-start/login.jp.mdx
index dc36af4..77b20e5 100644
--- crater-website/content/docs/user/quick-start/login.jp.mdx
+++ crater-website/content/docs/user/quick-start/login.jp.mdx
@@ -33,24 +33,24 @@ GPU クラスターのアクセス権をお持ちでない場合、以下のよ
 
 </Callout>
 
-アカウントが開通後、「ラボ」->「マイ情報」に新しいGPUクラスターへのアクセスボタンが表示されます。
+アカウントが開通後、「ラボ」->「マイ情報」に新しい GPU クラスターへのアクセスボタンが表示されます。
 
-![新しいGPUクラスターへのアクセス権](./images/new-access-to-gpu.webp)
+![新しい GPU クラスターへのアクセス権](./images/new-access-to-gpu.webp)
 
 プラットフォームに初めてログインする際は、このボタンを使ってアカウントをアクティベーションしてください。アクティベーション後、指導教員や研究室などの情報をこのプラットフォームに同期します。
 
-以降のログインでは、ACTアカウントを使ってCraterプラットフォームに直接ログインできます。
+以降のログインでは、ACT アカウントを使って Crater プラットフォームに直接ログインできます。
 
 ## よくある質問
 
-### 1. ラボ外からGPUクラスターにアクセスするにはどうすればいいですか？
+### 1. ラボ外から GPU クラスターにアクセスするにはどうすればいいですか？
 
-クライアントのダウンロード: https://vpn.buaa.edu.cn/portal，
+クライアントのダウンロード：https://vpn.buaa.edu.cn/portal，
 ログイン画面の下にある「クライアントをダウンロード」をクリックし、ポップアップで対応するオペレーティングシステムを選択してダウンロードしてください。
 
-### 2. VPNに接続できない場合の対処法:
+### 2. VPN に接続できない場合の対処法：
 
-ローカルのDNS解析が正しいか確認してください;
+ローカルの DNS 解析が正しいか確認してください;
 
 ```
 dig vpn.act.buaa.edu.cn
