diff --git crater-website/content/docs/admin/more/rdma.en.mdx crater-website/content/docs/admin/more/rdma.en.mdx
index 314cb53..d97fdd8 100644
--- crater-website/content/docs/admin/more/rdma.en.mdx
+++ crater-website/content/docs/admin/more/rdma.en.mdx
@@ -277,7 +277,7 @@ During the configuration of the RDMA Shared Device Plugin (referred to as Method
 >
 > we should update instructions and examples.
 
-So I read the documentation again and found a section called 「GPUDirect Workloads」 (inner OS: Are other installation methods not for GPU Workloads?)
+So I read the documentation again and found a section called [GPUDirect Workloads] (inner OS: Are other installation methods not for GPU Workloads?)
 
 Compared to Method 1, this method requires installing the DOCA driver, SR-IOV Device Plugin, Secondary network, Multus CNI, Container Networking plugins, IPAM plugin, where Multus CNI is a secondary network CNI in Kubernetes[^4].
 
