diff --git crater-website/content/docs/user/quick-start/user-account.jp.mdx crater-website/content/docs/user/quick-start/user-account.jp.mdx
index 3be4190..04cca0f 100644
--- crater-website/content/docs/user/quick-start/user-account.jp.mdx
+++ crater-website/content/docs/user/quick-start/user-account.jp.mdx
@@ -3,7 +3,7 @@ title: ユーザーとアカウント
 description: ユーザーが増えるにつれて、クォータに関する詳細な内容を整える可能性があります。
 ---
 
-ベータテスト期間中、一時的にユーザーとアカウントに関する完全なクォータ機能は開かれておらず、現在では各ユーザーの提出に上限は設定されていません。FIFOのルールに従ってキューに入れられることになるため、このセクションについてはご関心を持たなくてもかまいません。
+ベータテスト期間中、一時的にユーザーとアカウントに関する完全なクォータ機能は開かれておらず、現在では各ユーザーの提出に上限は設定されていません。FIFO のルールに従ってキューに入れられることになるため、このセクションについてはご関心を持たなくてもかまいません。
 
 将来的にユーザー数が増えるにつれて、クォータに関する詳細な内容を整える可能性があります。
 
